components/WorkoutType.js

```diff
@@ -0,0 +1,118 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components/native';
+
+export const WorkoutType = ({navigation}) => {
+
+  useEffect(()=> {
+    navigation.setOptions({headerShown:false});
+  });
+
+  const WorkoutTypesArray = [
+      { 
+        type: 'Yoga', 
+        color: '#588B8B',
+        image: require('../assets/yoga.png')
+      },
+      { 
+        type: 'Run', 
+        color: '#597799',
+        image: require('../assets/workout.png')
+      },
+      { 
+        type: 'Lift', 
+        color: '#5F91A3',
+        image: require('../assets/weightlift.png')
+      },
+      {
+        type: 'Walk', 
+        color: '#5FA390',
+        image: require('../assets/walking.png')
+      },
+      {
+        type: 'Rest Day', 
```

Comment:
Minor thing, but later when you concatenate the string - it adds up to "Just go away rest day" - which _do_ rhyme but it's good to think about if all data fits in the string your adding to :)

Reply:
Also, this array could be located in its own file and imported into this file - for some more structure in your project :)